metisp/workflows/metis/metis_lm_img_wkf.py

```diff
@@ -30,3 +38,56 @@
                     .with_meta_targets([SCIENCE])
                     .build())
 
+basic_reduction_sky = (task('metis_lm_img_basic_reduce_sky')
+                    .with_recipe('metis_lm_img_basic_reduce')
+                    .with_main_input(lm_raw_sky)
+                    .with_associated_input(lingain_task)
+                    .with_associated_input(dark_task)
+                    .with_associated_input(flat_task)
+                    .with_meta_targets([SCIENCE])
+                    .build())
+
+basic_reduction_std = (task('metis_lm_img_basic_reduce_std')
+                    .with_recipe('metis_lm_img_basic_reduce')
+                    .with_main_input(lm_raw_std)
+                    .with_associated_input(lingain_task)
+                    .with_associated_input(dark_task)
+                    .with_associated_input(flat_task)
+                    .with_meta_targets([SCIENCE])
+                    .build())
+
+background_sci_task = (task('metis_lm_img_background_sci')
+                    .with_recipe('metis_lm_img_background')
+                    .with_main_input(basic_reduction_sci)
+                    .with_associated_input(basic_reduction_sky)
+                    .with_meta_targets([SCIENCE])
+                    .build())
+
+background_std_task = (task('metis_lm_img_background_std')
+                    .with_recipe('metis_lm_img_background')
+                    .with_main_input(basic_reduction_std)
+                    .with_associated_input(basic_reduction_sky)
+                    .with_meta_targets([SCIENCE])
+                    .build())
+
+standard_flux_task = (task('metis_lm_img_standard_flux')
+                    .with_recipe('metis_lm_img_std_process')
+                    .with_main_input(background_std_task)
+                    .with_associated_input(fluxstd_catalog)
+                    .with_meta_targets([SCIENCE])
+                    .build())
+
+img_calib = (task('metis_lm_img_calib')
+             .with_recipe('metis_lm_img_calibrate')
+             .with_main_input(background_sci_task)
+             .with_associated_input(standard_flux_task)
+             .with_associated_input(distortion_task)
+             .with_meta_targets([SCIENCE])
+             .build())
+             
+img_coadd = (task('metis_lm_img_coadd')
+             .with_recipe('metis_lm_img_sci_postprocess')
```

Comment:
I strongly suggest keeping the task names as close to the recipe names as possible. Other names only add confusion.